src/components/SearchBar.tsx

```diff
@@ -39,7 +60,7 @@ const SearchBar = ({ onSearch }: SearchBarProps) => {
     <div className="flex w-full justify-between gap-4">
       <input
         type="text"
-        value={searchTerm}
+        value={inputValue}
         onChange={handleChange}
         onKeyDown={handleKeyDown}
         className="bg-white/10 py-2 pl-4 rounded-2xl flex-grow mr-4 text-base text-black placeholder:text-gray-400 border border-gray-300 focus:outline-none focus:ring-2 focus:ring-blue-400"
```

Comment:
Refactor: Improve className readability with twMerge grouping

- Better readability: Logical grouping of styles by purpose (layout, spacing, typography, states)
- Easier maintenance: Finding and modifying specific style groups is faster
- Conflict resolution: twMerge automatically handles duplicate/conflicting Tailwind classes
- Scalability: Easy to add responsive modifiers or conditional styles later

This approach makes the code more maintainable and follows modern Tailwind best practices.

```suggestion
className={twMerge(
  // Background & Layout
  'bg-white/10 flex-grow',
  // Spacing & Shape
  'py-2 pl-4 mr-4 rounded-2xl',
  // Typography
  'text-base text-black placeholder:text-gray-400',
  // Border & States
  'border border-gray-300',
  'focus:outline-none focus:ring-2 focus:ring-blue-400'
)}
```